tcms/testruns/tests/test_admin.py

```diff
@@ -62,6 +62,11 @@ class TestTestExecutionStatusAdmin(LoggedInTestCase):
     def setUpTestData(cls):
         super().setUpTestData()
         initiate_user_with_default_setups(cls.tester)
+        statuses = TestExecutionStatus.objects.all()
+        cls.negative = []
+        for status in statuses:
+            if status.weight < 0:
+                cls.negative.append(status.pk)
```

Comment:
You don't need to iterate over all statuses one by one. You can issue a query and then use `list(query_set)` to turn everything to a list.